event/handler.go

```diff
@@ -281,6 +283,55 @@ var SortFilter = func(ctx context.Context, handler *ExportHandler, event *Filter
 	}
 }
 
+var CreateFilterForAll = func(ctx context.Context, handler *ExportHandler, event *FilterSubmitted, isPublished bool) (*observation.DimensionFilters, error) {
+	// Get the names of the dimensions for the DatasetID
+	dimensions, err := handler.datasetAPICli.GetVersionDimensions(ctx,
+		"", // userAuthToken
+		handler.serviceAuthToken,
+		"", // collectionID
+		event.DatasetID, event.Edition, event.Version)
+
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
A couple of small formatting gripes, I think scopes should be closed on the same column they're open including for parenthesis when calling functions.. referring to the calls on lines 288 (above) and 300 (below). Also you probably don't want the blank line between creating and checking the error above.
Usually the first argument will start on the second line too unless you're opening a new scope in the arguments.

Reply:
Thanks for review.
I suspect you meant line 308.
Gripes fixed.